core/modules/qproc/QuerySession.cc

```diff
@@ -356,21 +354,25 @@ void QuerySession::_applyConcretePlugins() {
 }
 
 /// Some code useful for debugging.
-void QuerySession::_showFinal(std::ostream& os) {
-    // Print out the end result.
-    query::QueryTemplate par = _stmtParallel.front()->getTemplate();
-    query::QueryTemplate mer = _stmtMerge->getTemplate();
-
-    os << "\tQuerySession::_showFinal() : parallel: " << par.dbgStr() << std::endl;
-    os << "\tQuerySession::_showFinal() : merge: " << mer.dbgStr() << std::endl;
+std::string QuerySession::toString() {
```

Comment:
I didn't knew that, thanks. So you're right toString() isn't required for logging. Do you want me to change it here, or in an other ticket?